docs/Backup_and_restore_FARE.md

```diff
@@ -0,0 +1,115 @@
+# Backup and restore FARE
+
+The backup is composed of three parts:
+- Databse where are stored all the metadata and the users
+- Elasticsearch which is used to index the records and search them
+- Files which should be stored in a shared volume
+
+## 0. Shared volumes
+
+Data are shared between containers and host, they was added in the `docker-compose.full.yml` file and are organazied as follow:
+- To share the dump of the metadata between the `fare-invenio_db_1` container and the host, it was added  `/fare_shared_data/fare_db:/backup_fare_db` as volume in the `db` section
+- To share the dump of the indexes between the `fare-invenio_es_1` container and the host, it was added `/fare_shared_data/fare_es:/usr/share/elasticsearch/backup_fare_es` as volume in the `es` section
+- `/fare_shared_data/fare_files:/opt/invenio/instance/data/` was added in the `web-ui` section, to share the files uploaded in the platform
+
+## 1. Database
+The dump of the metadata and the users can be done with this command:
+```
+docker exec fare-invenio_db_1 pg_dumpall -U fare > ~/fare_shared_data/fare_db/backup_fare_db.sql

```

Comment:
Are we sure `fare` is the user that should be public? The DB user will be stored in the .env file

Reply:
You are right, i missed it